### ISSUE TITLE
Report Table: very difficult to discern column header highlight during keyboard navigation

### ISSUE THREAD

**Issue by jeffstieler** [woocommerce/woocommerce-admin#1610]
**Describe the bug**

It's very difficult to see the report table column header highlight when keyboard navigating. It's impossible to see in IE.

Found as part of #1525.

**To Reproduce**

1. Go to any report
2. Tab to the report table, tab through column headers
3. Verify that the sort caret displays, but otherwise it's extremely difficult to determine which column you're on

**Expected behavior**

Obvious visual indication of column header selection.

**Screenshots**

![screen shot 2019-02-18 at 12 58 40 pm](https://user-images.githubusercontent.com/63922/52973957-29febe00-337d-11e9-8019-5eac793d7aef.png)

SKU is highlighted - can you tell?

**Desktop (please complete the following information):**

* OS: Mac OS 10.14
* Browser: tested in Chrome, FF, IE

**Comment by LevinMedia** [comment]
heya @jeffstieler, so, in addition to what's already going on there, can we add the focus styles we're using on the summary numbers? 

It adds a dark black outline to the focused element.
 
<img width="1719" alt="Screen Shot 2019-07-24 at 11 54 35 AM" src="https://user-images.githubusercontent.com/4500952/61820529-06e3a900-ae0a-11e9-92cd-cc7a3a074c02.png">

Coupons is in focus in the screen shot above

**Comment by psealock** [comment]
We're now inheriting focus styles from wp component's Button.
